include/z64environment.h

```diff
@@ -72,26 +136,26 @@ typedef struct {
     /* 0x14 */ char unk_14[0x01];
     /* 0x15 */ u8 skyboxDisabled;
     /* 0x16 */ u8 sunMoonDisabled;
-    /* 0x17 */ u8 unk_17; // currentWeatherMode for skybox? (prev called gloomySky)
-    /* 0x18 */ u8 unk_18; // nextWeatherMode for skybox?
-    /* 0x19 */ u8 unk_19;
-    /* 0x1A */ u16 unk_1A;
+    /* 0x17 */ u8 skyboxConfig; // only used outdoors
+    /* 0x18 */ u8 nextSkyboxConfig; // only used outdoors
+    /* 0x19 */ u8 changeSkyState;
+    /* 0x1A */ u16 changeSkyTimer;
     /* 0x1C */ char unk_1C[0x02];
-    /* 0x1E */ u8 indoors; // when set, day time has no effect on lighting
-    /* 0x1F */ u8 unk_1F; // outdoor light index
-    /* 0x20 */ u8 unk_20; // prev outdoor light index?
-    /* 0x21 */ u8 unk_21;
-    /* 0x22 */ u16 unk_22;
-    /* 0x24 */ u16 unk_24;
+    /* 0x1E */ u8 indoors; // when true, day time has no effect on lighting
+    /* 0x1F */ u8 lightConfig; // only used outdoors
+    /* 0x20 */ u8 nextLightConfig; // only used outdoors
+    /* 0x21 */ u8 changeLights;
+    /* 0x22 */ u16 changeLightTimer;
+    /* 0x24 */ u16 outdoorChangeDuration; // total time to change skybox and light configs
```

Comment:
I'm not sure why this one specifically is named `outdoorChange`
I don't think it's useful to put that in the name, `lightConfig` is the main thing about this light stuff and doesn't have it

Reply:
i need to describe whats changing somehow, changeDuration is not enough imo. if you have any ideas feel free

Reply:
I was thinking the `change` prefix is enough

if it's really a worry `changeSkyboxAndLightDuration` I guess

Reply:
`changeConfigsDuration`?